server/src/service/BoardService.js

```diff
@@ -40,6 +40,20 @@ export class BoardService extends BaseService {
         return { myBoards, invitedBoards };
     }
 
+    @Transactional()
+    async getBoardIdsByUserId(userId) {
+        const boards = await this.boardRepository
+            .createQueryBuilder('board')
+            .select('board.id', 'id')
+            .leftJoin('board.invitations', 'invitation')
+            .where(`board.creator_id=:userId or user_id=:userId`, { userId })
```

Comment:
```suggestion
            .where(`board.creator_id=:userId or invitation.user_id=:userId`, { userId })
```
invitation에 대한 조건이 빠진듯 해요

Reply:
commit 600a091
위의 리뷰 내용과 해당 리뷰 내용에 따라 코드 수정하였습니다:)